assets/components/cards/render/CardFooter.jsx

```diff
@@ -19,6 +21,12 @@ CardFooter.propTypes = {
     pictureAvailable: PropTypes.bool,
     source: PropTypes.string,
     versioncreated: PropTypes.string,
+    audioIf: PropTypes.array,
```

Comment:
What does `audioIf` and `videoIf` stand for? A more descriptive naming convention would help other developers know what it is used for, for example `pictureAvailable` tells us that this property defines if a picture is available or not which also tells us it's value is a boolean.